tests/docker/Makefile

```diff
@@ -5,9 +5,9 @@ ifeq ($(PYTHON),)
 PYTHON := python3.7
 endif
 
-.PHONY: all $(DOCKER_TARGETS) run_perforce
+.PHONY: all $(DOCKER_TARGETS) run_perforce perforce
```

Comment:
Please modify the first line instead

Reply:
As already explained before, modification of first line will create images 'perfoce-python3.6', 'perforce-python3.7', 'perforce-python3.8' and 'perforce-python3.9', and I can't think of a reason for doing so.